ntbs-service/wwwroot/css/_colors.scss

```diff
@@ -1,47 +1,58 @@
-$phe-red: #822433;
-$phe-red-dull: #610925;
-$phe-navy: #002776;
-$phe-yellow: #eaab00;
+$dark-blue: #003B5C;

```

Comment:
Is this a ukhsa colour? If so, we should say so. If not, where did it come from?

Reply:
It used to be PHE-navy, but I hadn't realised this colour was called 'midnight' in the Rebrand docs. I'll change it and double check if there are any others